lib/util/util.js

```diff
@@ -284,7 +284,10 @@ const Util = {
             }
         }
         // check if project config version is outdated compared to user's mcdev version
-        if (!properties.version || semver.gt(packageJsonMcdev.version, properties.version)) {
+        if (
+            !properties.version ||
+            ![null, 'patch'].includes(semver.diff(packageJsonMcdev.version, properties.version))
+        ) {
```

Comment:
like above, here we need the old solution combined with the new to check if the project's config file (in git) is outdated compared to my app's version.
yes, one could argue that here one check is sufficient cause otherwise we'd have exited earlier but that's not intuitive enough while reading the code

Reply:
resolved in discussion